.golangci.yaml

```diff
@@ -12,7 +12,8 @@ run:
 # output configuration options
 output:
   # colored-line-number|line-number|json|tab|checkstyle|code-climate, default is "colored-line-number"
-  format: tab
+  formats:
```

Comment:
Please at least reference similar PRs in other repos. Or better yet apply commits from @cthulhu-rider (like in https://github.com/nspcc-dev/rfc6979/pull/8), they have perfect descriptions.

Reply:
Updated